api/config/bifrost/config_request.go

```diff
@@ -32,7 +32,7 @@ func DefaultConfigRequest() *ConfigRequest {
 	c.V1.Sys.Log.MaxErrorLogsPerSecond = w.Int32(1000)
 
 	c.V1.Sys.Sql.Timeout = w.Int32(5000)
-	c.V1.Sys.Sql.PoolInitSize = w.Int32(20)
+	c.V1.Sys.Sql.PoolInitSize = w.Int32(10)
```

Comment:
We may want to leave InitSize == MaxSize. We've definitely hit issues in the past with the pool scaling behavior.

Reply:
hey @stevendanna do you have any pointers to past discussions or test results that discuss these issues?

Reply:
Looking for them now.  I'm suspicious that these issues were never fully investigated but nonetheless became part of the "common wisdom."

Because the queue immediately rejects connections when it is out of connections and _then_ creates a new connection, I think the init=max behavior provided more comprehensible semantics before we had the request queue.

Reply:
one more thing I thought of -  we should also adjust erchef's `c.V1.Sys.Authz.Pool*` down to match a bit closer to this.

Reply:
I looked a bit at this in the morning and really, _all_ of the queues can probably use a second look. I feel like we don't *actually* want to start 100 ES connections, for example.